tnc/data_handler.py

```diff
@@ -126,6 +123,57 @@ def __init__(self) -> None:
 
         self.transmission_timeout = 360  # transmission timeout in seconds
 
+        # Dictionary of functions and log messages used in process_data
+        # instead of a long series of if-elif-else statements.
+        self.rx_dispatcher = {
```

Comment:
Nice! But we need to be carefully with older Python Versions. I'm not sure if this is working for older versions like 3.6 for example? We shouldn't focus that much on outdated software, but a lot of LTS operating systems are still using 3.6, I think.

Reply:
I'll specifically check on Python 3.6 but it's not a new feature - everything in Python is an object, including functions.

Reply:
Yes, no worries. I'm just not exactly sure, so better checking twice. But if you're saying it's okay, then we can merge it 👍

Reply:
Nope, there are a few problems with Python 3.6...  Fixing... :)

Reply:
Fixed.